src/hapi/pipelines/database/conflict_event.py

```diff
@@ -34,70 +34,78 @@ def populate(self):
         errors = set()
         for dataset in self._results.values():
             dataset_name = dataset["hdx_stub"]
+            # TODO: once unit has been recoded remove filter and error
+            if dataset_name.startswith("colombia"):
```

Comment:
This would be better in a YAML configuration, with a key name like excluded_datasets